.github/scripts/cleanspell.pl

```diff
@@ -70,7 +70,7 @@
         $_ =~ s/curl_ws_(send|recv|meta)//g;
         $_ =~ s/curl_url_(dup)//g;
         $_ =~ s/libcurl-env//g;
-        $_ =~ s/(^|\W)((tftp|https|http|ftp):\/\/[a-z0-9\-._~%:\/?\#\[\]\@!\$&'()*+,;=]+)//gi;
+        $_ =~ s/(^|\W)((tftp|https|http|ftp|ipfs|ipns):\/\/[a-z0-9\-._~%:\/?\#\[\]\@!\$&'()*+,;=]+)//gi;
```

Comment:
this isn't necessary, the cleanspell script only runs for libcurl man pages and IPFS is not a libcurl feature at this point

Reply:
Reverted.